cpu/nrf52/Makefile.include

```diff
@@ -1,5 +1,11 @@
 export CPU_ARCH = cortex-m4f
 export CPU_FAM  = nrf52
 
+# Export internal ROM alignment and slot sizes for bootloader support
+export IMAGE_ALIGN = 8
+export SLOT0_SIZE = 0x8000
+export SLOT1_SIZE = 0x3C000
+export SLOT2_SIZE = 0x3C000
```

Comment:
But I would be o.k. to postpone the fix of this to a follow up PR and search for a more generic solution for 'slicing' the ROM.